README.md

```diff
@@ -1,27 +1,92 @@
 # <div align="left"><img src="img/rapids_logo.png" width="90px"/>&nbsp;cuGraph - GPU Graph Analytics</div>
 
-[![Build Status](http://18.191.94.64/buildStatus/icon?job=cugraph-master)](http://18.191.94.64/job/cugraph-master/)  [![Documentation Status](https://readthedocs.org/projects/cugraph/badge/?version=latest)](https://docs.rapids.ai/api/cugraph/nightly/)

```

Comment:
I actually find this useful (when it is working...). Also, cuDF and cuML have it. Is there something preventing cuGraph from using this?

Reply:
it has not been working so I dropped it for now.  We can debug the issue and get it working on v0.8